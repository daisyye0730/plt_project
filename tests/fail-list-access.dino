int main() {
    List(int, 3) myli;
    string a;

    myli = [2, 3, 4];
    a = myli[2];
}