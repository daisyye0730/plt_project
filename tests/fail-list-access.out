Fatal error: exception Failure("illegal assignment string = int in a = myli[2]")
